app/src/components/common/layout/header/index.tsx

```diff
@@ -105,27 +108,74 @@ const HeaderContainer: React.FC<RouteComponentProps> = (props: RouteComponentPro
 
   const { history, ...restProps } = props
   const [isModalOpen, setModalState] = useState(false)
-
-  // hide connect button if country is blacklisted, or the info isn't available yet
-  const hideConnectButton = IS_CORONA_VERSION && (isBlacklistedCountry === null || isBlacklistedCountry === true)
+  const [isAdBlockDetected, setIsAdBlockDetected] = useState(false)
+
+  useEffect(() => {
```

Comment:
Let's move this to a custom hook, so that we can do:

```
const isAdBlockDetected = useDetectAdblock()
````

No need to make a new module for it since it will only be used in this file.